Applications/Spire/Source/Ui/ScrollBar.cpp

```diff
@@ -236,6 +236,7 @@ void ScrollBar::update_thumb() {
       style.get(Any()).set(PaddingLeft(m_thumb_position));
     }
   });
+  update();
```

Comment:
What does this do?

Reply:
Here I am trying to solve the bug of the ScrollBar. When enough ListItems are added into the ListView and the vertical scroll bar becomes visible, the thumb is not draggable even though the ListItems can move with the mouse dragging.

I will remove it and report the issue in Asana.

Reply:
You can keep this in here, just wasn't sure what it was doing.